community/kernel/src/main/java/org/neo4j/kernel/api/CountsRead.java

```diff
@@ -19,7 +19,7 @@
  */
 package org.neo4j.kernel.api;
 
-interface CountsRead
+public interface CountsRead
```

Comment:
Unnecessary. The only places where you use this seems to be in order to access the constants, and you can access those through `org.neo4j.kernel.api.ReadOperations`.